calendar-admins.members.txt

```diff
@@ -0,0 +1 @@
+kschriek@gmail.com
```

Comment:
```suggestion
kschriek@gmail.com
yliu989@bloomberg.net
```

Reply:
hey Dan, would you please add my email, josh.bottum@arrikto.com

Reply:
I am Yuzhui not Dan lol, but I added yours.

Reply:
Hi Yuzhui, my email is jtf.github@gmail.com. Can you add this for me?
Thanks

Reply:
Added :) @karlschriek to review.

Reply:
@yuzliu did Josh also volunteer to be a calendar admin? If so I also need to add him to the OWNERS file

Reply:
Hi @jbottum, just want to confirm that you want to be the calendar admin right?

Reply:
Please alphabetize the emails

Reply:
I've left of @jbottom for now, can be added later if needed, let's move this PR forward